setup.cfg

```
@@ -56,7 +56,7 @@ python_requires = >=3.7
 include_package_data = True
 install_requires =
     metomi-rose==2.3.*
-    cylc-flow>8.3.2,<8.4
+    cylc-flow>8.3.6,<8.4
```

Whoops, just spotted, this would have excluded the 8.3.6 release!

```suggestion
    cylc-flow>8.3.5,<8.4
```